core/domain/subtopic_page_domain_test.py

```diff
@@ -268,12 +268,6 @@ def setUp(self):
             subtopic_page_domain.SubtopicPageContents
             .create_default_subtopic_page_contents())
 
-    def _assert_validation_error(self, expected_error_substring):
```

Comment:
Why delete? Please justify.

Reply:
We are not using `_assert_validation_error` in this test class